packages/legend-application-studio/src/components/editor/aux-panel/Problems.tsx

```diff
@@ -64,7 +58,8 @@ const ProblemItem = observer((props: { problem: Problem }) => {
         </div>
         {problem.sourceInformation && (
           <div className="auxiliary-panel__problem__source">
-            {editorStore.isInGrammarTextMode &&
+            {editorStore.graphEditorMode.mode ===
```

Comment:
potentially if we have a place in rendering code layer to handle differentiating codepath, this is a good candidate to add there, together with all places we have to use `editorStore.graphEditorMode.mode` since although we decompose well at the state layer, our rendering is still messy